main.nf

```diff
@@ -40,24 +25,18 @@ params.rsem_index   = Workflow.getGenomeAttribute(params, 'rsem')
 params.salmon_index = Workflow.getGenomeAttribute(params, 'salmon')
 
 ////////////////////////////////////////////////////
-/* --         PRINT PARAMETER SUMMARY          -- */
-////////////////////////////////////////////////////
-
-def summary_params = NfcoreSchema.paramsSummaryMap(workflow, params, json_schema)
-log.info NfcoreSchema.paramsSummaryLog(workflow, params, json_schema)
-log.info Workflow.citation(workflow)
-log.info Utils.dashedLine(params.monochrome_logs)
-
-////////////////////////////////////////////////////
-/* --          PARAMETER CHECKS                -- */
+/* --          VALIDATE PARAMETERS             -- */
 ////////////////////////////////////////////////////
 
+def json_schema = "$projectDir/nextflow_schema.json"
 Workflow.validateMainParams(workflow, params, json_schema, log)
 
 ////////////////////////////////////////////////////
 /* --        NAMED PIPELINE WORKFLOW           -- */
 ////////////////////////////////////////////////////
 
+def summary_params = NfcoreSchema.paramsSummaryMap(workflow, params, json_schema)
```

Comment:
This is for the email right? Maybe we can just call it directly where it's used. This is rather isolated now.

Reply:
Done [here](https://github.com/drpatelh/rnaseq/blob/f1f5e891d4bcfd9c45965857b33b3cc66f2c2ce6/workflows/rnaseq.nf#L11) and removed from `main.nf`